elasticdl/python/elasticdl/api.py

```diff
@@ -103,6 +104,28 @@ def predict(args):
     _submit_job(image_name, args, container_args)
 
 
+def clean(args):
+    if args.docker_image_repository and args.all:
+        raise ValueError(
+            "--docker_image_repository and --all cannot "
+            "be specified at the same time"
+        )
+    if not (args.docker_image_repository or args.all):
+        raise ValueError(
+            "Either --docker_image_repository or --all "
+            "needs to be configured"
+        )
+    repository = (
+        args.docker_image_repository if args.docker_image_repository else ""
```

Comment:
`docker_image_repository` already defaults to `""` so this line doesn't seem necessary?

Reply:
Good catch. Yes, we can directly use `docker_image_repository` here. I introduced `--all` here to make sure users really want to clean up all images. `docker_image_repository` = ""  also means `--all`.